clean.cmd

```diff
@@ -0,0 +1,177 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
Is there a value on doing clean source or clean tools? usually if I delete the tools directory I delete everything else to be consistent with the new versions that I have, and I always clean the whole tree not just what is under src.
I could only leave clean all and the rest leave it to the run tool (cleaning bin, packagesdir and packagesCache)